packages/yoga/src/DropdownInput/web/DropdownInput.jsx

```diff
@@ -0,0 +1,317 @@
+import React, { useRef, useState } from 'react';
+import PropTypes, { bool } from 'prop-types';
+import { theme, Box, Icon, Text } from '@gympass/yoga';
+import styled, { css } from 'styled-components';
+import { Check, ChevronUp, ChevronDown, Close } from '@gympass/yoga-icons';
+import { display, flexes } from '@gympass/yoga-system';
+import { useOnClickOutside } from '../../hooks';
+
+const Wrapper = styled.div`
+  ${({
+    full,
+    theme: {
+      yoga: {
+        components: { dropdowninput },
+      },
+    },
+  }) => `
+    position: relative;
+    width: ${full ? '100%' : `${dropdowninput.width}px`};
+  `}
+`;
+
+const Container = styled.div`
+  ${({
+    theme: {
+      yoga: {
+        components: { dropdowninput },
+      },
+    },
+  }) => `
+    ${display}
+    ${flexes}
+    height: 52px;
```

Comment:
you can set this value in `DropdownInput.theme.js`

Reply:
✅